src/realm/query_expression.hpp

```diff
@@ -2279,6 +2410,11 @@ class Columns<Link> : public Subexpr2<Link> {
         return LinkCount(m_link_map);
     }
 
+    LinkCount size() const
+    {
+        return LinkCount(m_link_map);
```

Comment:
Either we should use the existing name, `count`, on the other column types, or we should replace `count` with `size`. Introducing a different name everywhere because you don't like the existing one doesn't seem ideal.